server/lib/cache.js

```diff
@@ -15,7 +15,7 @@ exports.register = function (server, options, next) {
     server.expose('set', client.set.bind(client));
     server.expose('drop', client.drop.bind(client));
 
-    client.start(next);
+    client.start().then(next).catch(next);
```

Comment:
catbox client start changed to being promise-based after v7 (oops) hopefully this solves the stall when connecting to rediscover